book/doctrine.rst

```diff
@@ -979,126 +1060,66 @@ To relate the ``Category`` and ``Product`` entities, start by creating a
             </entity>
         </doctrine-mapping>
 
-First, since a ``Category`` object will relate to many ``Product`` objects,
-a ``products`` array property is added to hold those ``Product`` objects.
-Again, this isn't done because Doctrine needs it, but instead because it
-makes sense in the application for each ``Category`` to hold an array of
-``Product`` objects.
+Doctrine does not *require* that the "one" side of a one-to-many relationship
+hold a collection of its "many" related entities. But in the context of our
```

Comment:
Why? The verb refers to the "the one side" subject of this sentence, doesn't it? And this is in its singular form.

We could also change it to "Doctrine does not require the "one" side of a one-to-many relationship to hold [...]". What do you think?